### PR TITLE
"Adding builds/files to my game page"

### PR DESCRIPTION
**Pull request by indiegamehunt**

### CODE REVIEW

File: add-build/index.md [hunk 1]
```diff
@@ -0,0 +1,15 @@
+## Adding builds/files to my game page
+
+Once your words and images have convinced someone to try your game, you have to give them something to play. If your game is far enough along in development that you have some sort of playable build (alpha, beta, prototype, etc.), go ahead and upload it. If your game's development status is "Complete" or "Canceled" then you are required to upload a build.
+
+To add a game build or another file, go to your [developer dashboard](http://gamejolt.com/dashboard/) and select a game from under "Your Games". You will be taken to the game's management page. Visit the "Builds & Files" tab and click "Add Package" to begin.
```
Comment (MaskofThoro) [inline add-build/index.md]: there is so much more to this, check out issue #56

File: add-build/index.md [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+
+You will need to enter a title and a description for each package you create. After you create the package, you can then add builds and files.
```
Comment (MaskofThoro) [inline add-build/index.md]: we should include help about adding builds and files here

**Comment by indiegamehunt** [comment]
This needs rewriting based on package/build info

File: add-build/index.md (hunk 1 above)
Comment (indiegamehunt) [inline add-build/index.md]: Rewrite underway.

**Comment by indiegamehunt** [comment]
Rewritten and ready for review.

**Pull request merged**